extension/parquet/parquet_float16.cpp

```diff
@@ -39,7 +43,8 @@ float Float16ToFloat32(const uint16_t &float16_value) {
 		float32_value = (sign << 31) | ((exponent + (127 - 15)) << 23) | (fraction << 13);
 	}
 
-	return *reinterpret_cast<float *>(&float32_value);
+	memcpy(&out, &float32_value, sizeof(out));
```

Comment:
Maybe we can use `Load<float>` here instead?

Reply:
Done, haven't tested locally.